vmdb/lib/workers/schedule_worker.rb

```diff
@@ -233,6 +216,23 @@ def schedules_for_scheduler_role
       @queue.enq :storage_scan_timer
     end
 
+    schedules_for_ems_refresh
+  end
+
+  def schedules_for_ems_refresh
+    config = VMDB::Config.new("vmdb").config.fetch(:ems_refresh, {})
+
+    ExtManagementSystem.leaf_subclasses.each do |klass|
```

Comment:
Is leaf_descendants better, since descendants gives you everything in the subtree of objects?